src/client/gui/lib/catalogue/launch_form.dart

```diff
@@ -83,6 +84,69 @@ class _LaunchFormState extends ConsumerState<LaunchForm> {
       width: 360,
     );
 
+    final zonesAsync = ref.watch(zonesProvider);
```

Comment:
Could you move watching the provider to the top, where all the other watches are happening? Also, there you could add a `final availableZones = zones.where((z) => z.available).toList();`, since you need that in multiple places.